src/model/extension.ts

```diff
@@ -135,6 +136,45 @@ export class DataFilterExtension extends BaseExtensionModel {
   }
 }
 
+export class PathStyleExtension extends BaseExtensionModel {
+  static extensionType = "path-style";
+
+  extensionInstance: _PathStyleExtension;
+
+  constructor(
+    model: WidgetModel,
+    layerModel: BaseLayerModel,
+    updateStateCallback: () => void,
+  ) {
+    super(model, updateStateCallback);
+
+    const dash = this.model.get("dash");
+    const highPrecisionDash = this.model.get("high_precision_dash");
+    const offset = this.model.get("offset");
+    this.extensionInstance = new _PathStyleExtension({ dash });
```

Comment:
I think you want to pass in `highPrecisionDash` and `offset` as well. We should only pass in defined parameters though, so you should probably use an `isDefined` cantonation like

```
new _PathStyleExtension({
      ...(isDefined(dash) && { dash }),
      ...(isDefined(highPrecisionDash) && { highPrecisionDash }),
      ...(isDefined(offset) && { offset }),
```
in this case, if any of those values are `undefined`, then it won't get passed in to the constructor

Reply:
Cheers. Done. Also added the import for `isDefined`. I wasn't sure what to pass in to the constructor and kept it as [`dash` only as that was in the `deck.gl` API reference](https://deck.gl/docs/api-reference/extensions/path-style-extension#constructor).